### DISCUSSION TITLE
Upgrade to .NET 8 or stay on .NET 6 until end-of-support

### DISCUSSION THREAD

**Discussion by christianhelle** [christianhelle/refitter#214] [Polls]
.NET 8.0 is scheduled for release next week (November 14) and I'm usually really quick to jump on the latest and greatest bandwagon but it has been brought to my attention that I have users who are constrained to LTS versions of .NET

Since .NET 8.0 has long-term support, I'm considering upgrading the Refitter CLI tool to target .NET 8

**Comment by christianhelle** [comment]
@kirides is .NET 8.0 a problem for you?

**Reply by kirides** [reply]
@christianhelle 

we use Refitter as a commandline application, not the source generator part, so for us it doesn't really matter that much.

I guess upping to NET8 is fine, unless Refit causes breaking changing with upgrades to 8.0.0
